src/OP2Internal.asm

```diff
@@ -12,7 +12,7 @@
 ; You must set the following to the load address of the DLL
 ; to ensure proper linking to the functions in Outpost2.exe
 ; *********************
-LoadAddr EQU 0x11000000
+LoadAddr EQU 0x14000000
```

Comment:
Uhh, was this intentional? I have a feeling it wasn't.

Reply:
Fix with a forced push that changed commit. My bust. It would be nice if we could manipulate LoadAddr without changing the value directly for easing repository work.

Reply:
I know. We need to come up with a workable solution, though I don't think that's going to be easy.